src/components/views/dialogs/PrototypeCreateGroupDialog.tsx

```diff
@@ -0,0 +1,220 @@
+                            "cannot be changed.",
+                        )}
+                    />
+                </span>
+            );
+        }
+
+        let helpText = (
+            <span className="mx_PrototypeCreateGroupDialog_subtext">
+                {_t("You can change this later if needed.")}
+            </span>
+        );
+        if (this.state.error) {
+            helpText = (
+                <span className="mx_PrototypeCreateGroupDialog_subtext mx_PrototypeCreateGroupDialog_subtext_error">
+                    {this.state.error}
+                </span>
+            );
+        }
+
+        let preview = <img src={this.state.avatarPreview} className="mx_PrototypeCreateGroupDialog_avatar" />;
+        if (!this.state.avatarPreview) {
+            preview = <div className="mx_PrototypeCreateGroupDialog_placeholderAvatar" />
+        }
+
+        return (
+            <BaseDialog
+                className="mx_PrototypeCreateGroupDialog"
+                onFinished={this.props.onFinished}
+                title={_t("What's the name of your community or team?")}
+            >
+                <form onSubmit={this.onSubmit}>
+                    <div className="mx_Dialog_content">
+                        <div className="mx_PrototypeCreateGroupDialog_colName">
+                            <Field
+                                value={this.state.name}
+                                onChange={this.onNameChange}
+                                placeholder={_t("Enter name")}
+                                label={_t("Enter name")}
+                            />
+                            {helpText}
+                            <span className="mx_PrototypeCreateGroupDialog_subtext">
+                                {/*nbsp is to reserve the height of this element when there's nothing*/}
+                                &nbsp;{communityId}
+                            </span>
+                            <AccessibleButton kind="primary" onClick={this.onSubmit} disabled={this.state.busy}>
+                                {_t("Create")}
+                            </AccessibleButton>
+                        </div>
+                        <div className="mx_PrototypeCreateGroupDialog_colAvatar">
+                            <input
+                                type="file" style={{display: "none"}}
+                                ref={this.avatarUploadRef} accept="image/*"
+                                onChange={this.onAvatarChanged}
+                            />
+                            <AccessibleButton onClick={this.onChangeAvatar} className="mx_PrototypeCreateGroupDialog_avatarContainer">
+                                {preview}
+                            </AccessibleButton>
+                            <div className="mx_PrototypeCreateGroupDialog_tip">
+                                <b>{_t("PRO TIP")}</b>
```

Comment:
I hope this is from an actual design... 😆

Reply:
it is, though copy review is slated for this week (this translates nicely in the UK world, but North Americans have a very different association with this phrasing).

![image](https://user-images.githubusercontent.com/1190097/91316133-b9a4da00-e775-11ea-8272-619df6f70cd7.png)

(yes, the designs don't look like the implementation - this is expected)

Reply:
quick copy review is changing this ftr